app/src/main/java/de/danoeh/antennapod/activity/MediaplayerActivity.java

```diff
@@ -443,7 +474,22 @@ protected void setupGUI() {
         setContentView(getContentViewResourceId());
         sbPosition = (SeekBar) findViewById(R.id.sbPosition);
         txtvPosition = (TextView) findViewById(R.id.txtvPosition);
+
+        SharedPreferences prefs = getSharedPreferences(PREFS, MODE_PRIVATE);
+        SharedPreferences.Editor editor = prefs.edit();
+        showTimeLeft = prefs.getBoolean(PREF_SHOW_TIME_LEFT,false);
+        Log.w("timeleft",showTimeLeft? "true":"false");
```

Comment:
Sorry, that wasn't supposed to stay there! it was for me to check the value when debugging, just used whatever android studio first suggested me :laughing:  
About the editor: is it wrong being instantiated here? wouldn't doing it onClick create an instance every time the user taps the time?